test/integration/connect/envoy/case-cfg-resolver-svc-redirect/setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# manually setup config entries
+echo '
+kind = "proxy-defaults"
+name = "global"
+config {
+  protocol = "http"
+}
+' | docker_consul config write -
+
+echo '
+kind = "service-defaults"
+name = "s1"
+protocol = "http"
+' | docker_consul config write -
+echo '
+kind = "service-defaults"
+name = "s2"
+protocol = "http"
+' | docker_consul config write -
+echo '
+kind = "service-defaults"
+name = "s3"
+protocol = "http"
+' | docker_consul config write -
+
+echo '
+kind = "service-resolver"
+name = "s2"
+redirect {
+  service = "s3"
+}
+' | docker_consul config write -
```

Comment:
I don't mind this, but isn't it easier to just add them as bootstrap entries in`*.hcl` files in the test case which will be included when consul starts? That's how I did other proxy setup stuff.

Reply:
I hadn't merged that other PR yet (unifying `structs` and `api` data massaging) when I wrote this and didn't want to also try to debug anything in that.

That said, let me see if this test passes if i switch it back over to just straight bootstrap entries now that I know what the real problem is.

Reply:
And that works. Updated.